assets/js/mobile-app-banner.js

```diff
@@ -1,28 +1,45 @@
-const isAlertsPage = () => {
-  const url = window.location.href;
-
-  return /\/alerts\/[\w.-]+/.test(url);
+const isAlertsPage = url => {
+  return /\/alerts/.test(url);
 };
 
-const isIPhone = () => {
-  return /iPhone/.test(navigator.userAgent);
+const isNotAndroid = () => {
+  return !/Android/.test(navigator.userAgent);
 };
 
-const isSchedulePage = () => {
-  const url = window.location.href;
+const isNotCommuterRail = url => {
+  return !/\/schedules\/CR-/.test(url);
+};
 
+const isSchedulePage = url => {
   return /\/schedules\/[\w.-]+\/line/.test(url);
 };
 
-const isStopPage = () => {
-  const url = window.location.href;
-
+const isStopPage = url => {
   return /\/stops\/[\w.-]+/.test(url);
 };
 
+const isTransitNearMePage = url => {
+  return /\/transit-near-me/.test(url);
+};
+
+const is = url => {
+  return (
+    isAlertsPage(url) ||
+    isSchedulePage(url) ||
+    isStopPage(url) ||
+    isTransitNearMePage(url)
+  );
+};
+
+const isNot = url => {
+  return isNotAndroid() && isNotCommuterRail(url);
+};
+
 // Show the mobile app banner
 export default function mobileAppBanner() {
-  if (isIPhone() && (isAlertsPage() || isSchedulePage() || isStopPage())) {
+  const url = window.location.href;
+
+  if (is(url) && isNot(url)) {
```

Comment:
lolololol at this naming

Reply:
I thought about `ises` and `isNots`...but the former probably isn't a word.